tests/integration/nm/checkpoint.py

```diff
@@ -127,11 +126,18 @@ def test_plugin_create_and_rollback_checkpoint(nm_plugin):
     assert len(checkpoints) == 0
 
 
+def _refresh_content(nmplugin):
+    context = nmplugin._ctx.context
+    if context:
+        while context.iteration(False):
+            pass
+
+
 def test_plugin_create_checkpoint_and_timeout(nm_plugin):
     plugin = nm_plugin
     plugin.create_checkpoint(timeout=1)
     time.sleep(2)
 
-    nm_plugin.refresh_content()
+    _refresh_content(plugin)
```

Comment:
Could you explain why this is need in test while the production code does not?

Reply:
The test code does not represent a real behaviour. On Nmstate if the user try to do rollback() and the checkpoint has expired, Nmstate will fail with the following error:

```
E           libnmstate.error.NmstateLibnmError: Checkpoint rollback failed: error=nm-manager-error-quark: checkpoint /org/freedesktop/NetworkManager/Checkpoint/1015 does not exist (12)
```

I've modified the integration test so `_refresh_content()` is not needed anymore.